src/App.js

```diff
@@ -1,5 +1,60 @@
+const { Console } = require("@woowacourse/mission-utils");
+const BaseballGame = require("./BaseballGame");
+const { COMMAND, COUNT } = require("./constants/game");
+
+const InputView = require("./view/InputView");
+const OutputView = require("./view/OutputView");
+
 class App {
-  play() {}
+  constructor() {
+    this.baseballGame = new BaseballGame();
+  }
+
+  play() {
+    OutputView.printStart();
+    this.start();
+  }
+
+  start() {
+    this.baseballGame.createAnswerNumbers();
+
+    this.readPlayerInput();
+  }
+
+  readPlayerInput() {
+    InputView.readPlayerInput(this.compareAnswer.bind(this));
+  }
```

Comment:
bind를 사용하게 되면 또다른 함수가 만들어지는 것과 같은 일이라, 그렇게 되면 안 쓰는 원래의 함수 + 변형된 함수 이렇게 두 개의 함수가 만들어지게 되어 메모리를 차지하게 만듭니다. Arrow 함수를 이용하면 bind를 쓰지 않고 this를 이용하며 기존의 함수를 이용할 수 있습니다👍

```suggestion
  readPlayerInput() {
    InputView.readPlayerInput((param) => this.compareAnswer(param));
  }
```

이 방법은 중간에 함수를 걸쳐가기 때문에 진입 비용이 조금 발생할 수는 있으나, 그 비용이 미치는 영향은 미미하다고 알고 있습니다. 알고 계시면 좋을 듯 합니당😊